avocado/utils/nvme.py

```
@@ -529,3 +530,121 @@
+    elif process.system(cmd, shell=True, ignore_status=True):
+        raise NvmeException(f"namespace {namespace} lock failed")
+    if is_drive_locked(namespace):
+        raise NvmeException(f"locking failed for {namespace}")
+
+
+def pexpect_cmd_execution(cmd, list_of_expect_sendline):
+    pexpect_handle = pexpect.spawn(cmd)
+    for expect, value in list_of_expect_sendline:
+        pexpect_handle.expect(expect, timeout=30)
+        pexpect_handle.sendline(value)
+        output = pexpect_handle.before.strip()
```

Ruff (0.12.2)

Local variable `output` is assigned to but never used

Remove assignment to unused variable `output`

(F841)